pages/apply.html

```diff
@@ -0,0 +1,380 @@
+
+<script type="text/javascript" src="../assets/js/jquery-1.11.3.min.js"></script>
+<script type="text/javascript" src="../assets/js/parsley.min.js"></script>
+
+<body>
+
+  <div id="main">
+
+    <header>
+      <h1>Apply to Join 18F</h1>
+      <img src="../assets/img/18f-logo.png" alt="18F Logo" class="logo" />
+    </header>
+
+    <script type="text/javascript">var submitted=false;</script>
+    <iframe name="hidden_iframe" id="hidden_iframe"
+    style="display:none;" onload="if(submitted) {
+      $(document).ready(function() {
+        window.location.href = './apply-thank-you.html';
+      })}"></iframe>
+
+    <form action="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSfbe4KAEqQFp07f8zfD6xWAfdFTAaZ04tpDQRJnNPeBHoW6RQ/formResponse" method="post"
+    target="hidden_iframe" onsubmit="submitted=true;">
+
+      Fields that are marked with <span class="required">*</span> are required.
+
+      <fieldset class="eligibility">
+
+        <h2><legend>Eligibility:<span class="required">*</span></legend></h2>
+        <p>Any individual who is not currently a GSA employee or contractor and who is a U.S. citizen, a non-citizen national of the U.S. or holds a valid green card is eligible to apply.</p>
+
+        <div class="usa-input">
+          <ul class="usa-unstyled-list">
+            <li>
+              <input type="radio" name="entry.1377828897" value="Verify that you meet the above eligibility requirements" id="group_1377828897_1" role="radio" class="ss-q-radio" aria-required="true">
+              <label for="group_1377828897_1">I, the applicant, verify that I meet the above eligibility requirements.</label>
+            </li>
+            <li>
+              <input type="radio" name="entry.1377828897" value="I, the applicant, verify that I do not meet the above eligibility and will not be completing the rest of this application." id="group_1377828897_2" role="radio" class="ss-q-radio" aria-required="true">
+              <label for="group_1377828897_2">I, the applicant, verify that I do not meet the above eligibility requirements and will not be completing the rest of this application.</label>
+            </li>
+          </ul>
+        </div>
+
+    
+        
+      </fieldset>
+
+      <fieldset class="contact-information">
+
+        <h2><legend>Contact Information:</legend></h2>
+
+        <div class="usa-input name">
+
+          <p>
+          <label class="ss-q-item-label" for="entry_1741264393">Name (First Middle Last) <span class="required">*</span></label>
+          <br/>
+            If no middle name (NMN), write "NMN" between your first and last name.
+          </p>
+            <input type="text" name="entry.1741264393" value="" class="ss-q-short" id="entry_1741264393" dir="auto" aria-label="Name (First Middle Last)  " aria-required="true" title="" required />
+        </div>
+
+        <div class="usa-input preferredname">
+          <label class="ss-q-item-label" for="entry_732113689">Preferred name</label>
```

Comment:
I strongly recommend we just have one name field, synonymous with `Preferred name`, and leave the legal question for a later phase of the process. This recommendation is because as far as I'm aware, there isn't a need to have someone's legal name at this point, the legal name question risks introducing unnecessary confusion into future communications, and I promise you it risks making some applicants uncomfortable to such a degree that they won't apply.

If we do proceed with two fields, however, I recommend making this one optional so your results don't get cluttered up with a lot of `N/A`s or the equivalent.